content/src/controller/Controller.ts

```diff
@@ -370,6 +370,15 @@ export class Controller {
     return '?' + nextQueryParams
   }
 
+  async getDeploymentsByHash(req: express.Request, res: express.Response) {
+    // Method: GET
+    // Path: /deployments/:contentHash
```

Comment:
Personally, I don't think that this path is the best for this use case, as the `:id` following the noun `deployments` should be the id of the deployment themselves and not the hash for content inside a deployment.

Let's use a different path, something maybe like `/deployments/content/:contentHash`? What do you think?

Reply:
Just my 2 cents, `/deployments/content/:contentHash` seems kind of weird to me, because we are not returning deployments nor content files. 

Just to keep adding options haha, maybe `/deployments/using/:contentHash`. Or maybe not under `/deployments` at all?

Reply:
Oh, I thought that we were returning a collection of deployments (or deployment ids) that's why it made sense to me.

Anyway, I'm not really comfortable using a verb in the path. I think I'd prefer a brand new noun, but it can't be `/content` either as we are not returning content haha.

Reply:
From the options I do prefer`/deployments/content/:contentHash`, but since we're not returning a content, who about something like `/deployments/content/active-entity/:contentHash`?

Reply:
Just to continue with the brainstorming. What about this one?
```/contents/:contentHash/active-entities```

Reply:
I thought something similar: `/available-content/:hashId/deployments` is available the same as active?

Reply:
Note: I'd avoid having two consecutive nouns in the path segments

Reply:
`/available-content/` is used for files that are "downloadable". It isn't related to entities or if they are active.

For example, if the server doesn't have garbage collection enabled, then the file will be available, even if there are no active entities referencing it.

Reply:
after a long discussion, we got to `/contents/:hashId/active-entity`